doc/source/merging.rst

```diff
@@ -817,6 +817,12 @@ equivalent:
     left.join(right, on=key_or_keys)
     pd.merge(left, right, left_on=key_or_keys, right_index=True,
           how='left', sort=False)
+          
+.. warning::
+
+   Joining on keys with duplicate values when joining large dataframes would cause severe memory overflow, sometimes freezes the
```

Comment:
This is not very useful. I think better is a simple statement that joining / merging on duplicate keys can cause a returned frame that is the multiplication of the row dimensions. Please show a small example as well.